pkg/controller/plan/adapter/base/doc.go

```diff
@@ -53,14 +56,20 @@ type Builder interface {
 	ResolvePersistentVolumeClaimIdentifier(pvc *core.PersistentVolumeClaim) string
 	// Conversion Pod environment
 	PodEnvironment(vmRef ref.Ref, sourceSecret *core.Secret) (env []core.EnvVar, err error)
-	// Create PersistentVolumeClaim with a DataSourceRef
-	PersistentVolumeClaimWithSourceRef(da interface{}, storageName *string, populatorName string, accessModes []core.PersistentVolumeAccessMode, volumeMode *core.PersistentVolumeMode) *core.PersistentVolumeClaim
-	// Add custom steps before creating PVC/DataVolume
-	PreTransferActions(c Client, vmRef ref.Ref) (ready bool, err error)
+	SupportsVolumePopulators() bool
+	// Build populator volumes
+	EnsurePopulatorVolumes(vmRef ref.Ref, annotations map[string]string, secretName string) (pvcNames []string, err error)
+	// Transferred bytes
+	PopulatorTransferredBytes(persistentVolumeClaim *core.PersistentVolumeClaim) (transferredBytes int64, err error)
+	// Set the populator PVC labels
+	SetPopulatorDataSourceLabels(vmRef ref.Ref, pvcs []core.PersistentVolumeClaim) (err error)
+	// Get the populator task name associated to a PVC
+	GetPopulatorTaskName(pvc *core.PersistentVolumeClaim) (taskName string, err error)
 	// Build LUN PVs.
 	LunPersistentVolumes(vmRef ref.Ref) (pvs []core.PersistentVolume, err error)
 	// Build LUN PVCs.
 	LunPersistentVolumeClaims(vmRef ref.Ref) (pvcs []core.PersistentVolumeClaim, err error)
+	// check whether the builder supports Volume Populators
```

Comment:
should be set on SupportsVolumePopulators it seems

Reply:
fixed it in the latest push :)